packages/html/package.json

```diff
@@ -50,5 +50,6 @@
   },
   "devDependencies": {
     "@types/react": "^18.0.2"
-  }
+  },
+  "main": "index.js"
```

Comment:
`main` is pretty important field. consider moving it up the tree for visibility. fields at the end of the file, tend to be ignored.

Reply:
moved